pkgs/development/libraries/libportal/default.nix

```diff
@@ -30,11 +30,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-wDDE43UC6FBgPYLS+WWExeheURCH/3fCKu5oJg7GM+A=";
   };
 
+  # TODO: remove on next release
```

Comment:
Can you please specify after which release version this patch will be included? Is there a link to a upstream issue or PR?

Reply:
Done.